packages/integration/src/bitbucket/core.ts

```diff
@@ -32,10 +32,11 @@ export async function getBitbucketDefaultBranch(
 
   const isHosted = resource === 'bitbucket.org';
   // Bitbucket Server https://docs.atlassian.com/bitbucket-server/rest/7.9.0/bitbucket-rest.html#idp184
+  // Changed branchUrl to support Atlassian Bitbucket v5.11.1 , which has different branchUrl format
   const branchUrl = isHosted
     ? `${config.apiBaseUrl}/repositories/${project}/${repoName}`
-    : `${config.apiBaseUrl}/projects/${project}/repos/${repoName}/default-branch`;
-
+    : `${config.apiBaseUrl}/projects/${project}/repos/${repoName}/branches/default`;
+    
```

Comment:
This is not what we agreed upon - it'll have to be more involved than this. It should first try the new format, and then if it gets specifically a 404 it should try the old format.

Reply:
Thanks. done  and tested ok.  Please review .

Reply:
Thanks! Could you also update the tests (core.test.ts) to include both these endpoints? We should have a case where the endpoint above results in 404 and the endpoint below is used.

Reply:
updated core.test.ts